README.md

```diff
@@ -69,10 +69,20 @@ workflows:
           context:
             - ai-llm-evals-orb-examples # Replace this with your context name
             - slack-notification-access-token # Replace this with your context name where SLACK_ACCESS_TOKEN is stored
+  custom-command-evals:
+    when: << pipeline.parameters.run-custom-command-evals >>
+    jobs:
+      - run-custom-command-evals:
+          context:
+            - ai-llm-eval-examples # Replace this with your context name
+            - slack-notification-access-token # Replace this with your context name where SLACK_ACCESS_TOKEN is stored
 ```
 
 ### Step 3. Select an evaluation platform
 
+The evals orb supports running evaluations on two evaluation platforms: [Braintrust](https://www.braintrustdata.com/) and [LangSmith](https://smith.langchain.com/).
+The orb also supports running a custom command without an evaluation platform and a path to a file with evals results.
```

Comment:
You say "without a path to a file with evals result". Just to make sure I'm clear - the user can still provide a path for eval results, but it's not required,  right?

Reply:
wrong words in my sentence. i will fix it. 

- supports without a platform
- support with or without a eval results location